### ISSUE TITLE
[Question] Channels inactive after IP change

### ISSUE THREAD

**Issue by otaviobps** [lightningnetwork/lnd#3896]
### Background

I have 67 open channels. Almost all of them were active. I then moved to another location and brought my node with me. It changed the IP address, and when I restarted it, the node announced the new IP to the network. Running `getinfo` returns the new IP address, everything is working fine.

However, now more than 40 channels are inactive. An owner of two of the channels I have contacted me asking why our channel is offline. I then started to debug why this happened. I noticed I wasn't connected to him, and trying to manually connect to him gives me `[lncli] rpc error: code = Unknown desc = dial tcp 94.107.123.223:9735: connect: connection refused`. His port is closed, so naturally I can't connect to him.

My question is, when I announced my new IP to the network, shouldn't the other peers connect to my node using my new IP? Is this the reason many of my channels are inactive? Is there a solution to it?

### Your environment

* version of `lnd`: `0.8.2-beta commit=v0.8.2-beta`
* which operating system (`uname -a` on *Nix): `Debian 9`
* version of `btcd`, `bitcoind`, or other backend: `bitcoind /Satoshi:0.18.1/`

### Steps to reproduce

Change the IP address and start the node

### Expected behaviour

All channels to be active

### Actual behaviour

Many channels are inactive

**Comment by cfromknecht** [comment]
> Running `getinfo` returns the new IP address, everything is working fine.

Try running `getnodeinfo <your_pubkey>`, does it show the correct URI in `"addresses"`? The URIs in `getnodeinfo` are actually pulled from your local routing table and reflect what you last _announced_. The URIs in `getinfo` display the _configured_ URIs, so it is possible they could be out of sync for various reasons.

> My question is, when I announced my new IP to the network, shouldn't the other peers connect to my node using my new IP? Is this the reason many of my channels are inactive? Is there a solution to it?

More than likely the issue is that the other 40 don't realize they need to sync your latest node announcement. This issue w/ node ann propagation has been known for some time, and we know of some ways to improve it but they aren't in place yet. For now the easiest solution is just to open a new, advertised channel (can be w/ anyone), which will cause those peers to resync info for the new channel and your latest node announcement in the process.

**Comment by wpaulino** [comment]
Node announcements are also rebroadcast every 24H of uninterrupted uptime.

**Comment by cfromknecht** [comment]
> Node announcements are also rebroadcast every 24H of uninterrupted uptime.

In this case I think the nodes have to be online at the time the announcement is broadcast to them. If you had unadvertised channels, opening a channel will cause them to sync your update immediately after coming back online.

**Comment by peername** [comment]
The same thing happened to us. We had an average of 30 inactive channels. After changing the IP address they increased to 60. It took 2-3 weeks to normalize back to 30. Eventually you will be fine, you need patience.

**Comment by otaviobps** [comment]
> Try running `getnodeinfo <your_pubkey>`, does it show the correct URI in "addresses"?

It does. Definitely the problem is not with my node. I advertise my domain to the network, which is configured with DDNS. When I restarted my node, my domain already had my new IP address.

The problem I noticed was that I was not able to reactivate channels with nodes that have their 9735 port closed, probably because I couldn't connect to them, and they didn't connect to me as well.

> Node announcements are also rebroadcast every 24H of uninterrupted uptime.

My node is online 24/7, and since I restarted it, more than 24h has passed. So this didn't help as well.
